docs/sources.md

```diff
@@ -4,6 +4,7 @@
 - [GARD](sources/gard.md)
 - [ICD10CM](sources/icd10cm.md)
 - [ICD10WHO](sources/icd10who.md)
+- [MEDGEN](sources/medgen.md)
```

Comment:
## `sources.md`
Added link to source. This is the only other file I updated other than `sources/gard.md` and `sources/medgen.md` in this PR. I did not set up MedGen to dynamically generate docs / etc. Just wanted to record about mapping assumptions.

Reply:
Is it somehow possible like we do in [OBO](https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/ontology/mondo.md?plain=1#L88) to have the bottom part of the docs custom and the top part autogenerated (see `make documentation` goal I made)